video/vdu.h

```diff
@@ -157,32 +157,112 @@ void VDUStreamProcessor::vdu_graphicsViewport() {
 // VDU 25 Handle PLOT
 //
 void VDUStreamProcessor::vdu_plot() {
-	auto mode = readByte_t(); if (mode == -1) return;
+	auto command = readByte_t(); if (command == -1) return;
+	auto mode = command & 0x07;
+	auto operation = command & 0xF8;
```

Comment:
"mode" here is the 0-7 codes that sit within each group of eight command block
the "operation" is the block of commands